cmake/VMCBuildProject.cmake

```diff
@@ -55,5 +55,5 @@ install(FILES
   "${PROJECT_BINARY_DIR}/VMCConfigVersion.cmake"
   DESTINATION ${CMAKE_INSTALL_LIBDIR}/VMC-${VMC_VERSION})
 
-install(EXPORT VMCTargets
+install(EXPORT VMCTargets NAMESPACE VMC::
```

Comment:
I don't see a need for adding the namespace, which problem are you trying to solve with this?

Reply:
Well, I find it an elegant way of being able to refer to such a library from another CMake file making it clear a certain library belongs to: VMC::someLib But ok, if you say the VMC package is pretty "small" and has finally essentially one library with a speaking name, we can remove the namespace.